packages/vuetify/src/util/helpers.ts

```diff
@@ -443,3 +431,12 @@ export function getSlotType<T extends boolean = false> (vm: Vue, name: string, s
   if (vm.$slots[name]) return 'normal'
   if (vm.$scopedSlots[name]) return 'scoped'
 }
+
+export function getSlot (vm: Vue, name = 'default', data?: object) {
```

Comment:
I think this one should have a return type

Reply:
It does 
![image](https://user-images.githubusercontent.com/9064066/54137625-f77c3a00-43eb-11e9-874b-cf037a304aec.png)